generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management-update.component.ts.ejs

```diff
@@ -324,22 +346,51 @@ _%>
         this.isSaving = false;
     }
 
-    <%_ if (selectableEntitiesType) { _%>
+    <%_ if (selectableEntitiesType) {
+        const idColumn = selectableEntitiesType === 'IAuthority' ? 'name' : 'id';
```

Comment:
Use otherEntityFieldName somehow?

Reply:
not sure if I should add more time here, as in this particular template we are working with a lot of variables anywhere. Maybe it's a concern to do it this way and adding another variable doesn't make it better. But I fear to make it in a different way, we should do a bigger work on here. WDYT?

Reply:
Yep, looks like we need a bigger refactoring here.